tests/epyccel/test_epyccel_functions.py

```diff
@@ -111,29 +111,32 @@ def f2(x):
 
 #------------------------------------------------------------------------------
 # Semantic error doesn't need testing in multiple languages
-def test_decorator_f3():
+def test_decorator_f3(language):
     @types('int [:]')
     def f3(x):
         from numpy import empty_like
         y = empty_like(x)
         y[:] = x - 1
         return y
 
-    with pytest.raises(PyccelError):
-        epyccel(f3)
+    f = epyccel(f3, language=language)
```

Comment:
You should still catch the expected error

Reply:
Actually you may only need to catch it for Fortran and C. Python should work out of the box as there is no wrapper